src/main/java/jenkins/plugins/publish_over_ssh/BapSshAlwaysRunPublisherPlugin.java

```diff
@@ -47,7 +48,7 @@ public BapSshAlwaysRunPublisherPlugin(final ArrayList<BapSshPublisher> publisher
     }
 
     @Override
-    protected boolean isBuildGoodEnoughToRun(final AbstractBuild<?, ?> build, final PrintStream console) {
+    protected boolean isBuildGoodEnoughToRun(final Run<?, ?> build, final PrintStream console) {
```

Comment:
@jglick @slide I am really aware about this guy, but it just justifies updating all plugins depending on "Publish over"